tests/caasp/stack_initialize.pm

```diff
@@ -29,6 +29,9 @@ sub run {
     assert_script_sudo "chown $testapi::username /dev/$testapi::serialdev";
     # Disable screensaver
     script_run "gsettings set org.gnome.desktop.session idle-delay 0";
+    # Update kubectl
+    assert_script_sudo "zypper -n up kubernetes-client", 300;
```

Comment:
Please, use zypper_call wrapper from lib/utils.pm

Reply:
zypper_call requires root user, so I would have to login, zypper_call, logout - this feels simpler and should have the same functionality

Reply:
Good point, missed that. We should consider extending zypper_call to run command using sudo.